AzureCalling/Controllers/AudioDeviceSelectionViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class AudioDeviceSelectionViewController: UIViewController, UITableViewDelegate {
```

Comment:
I don't think we need to hold `audioDeviceSelectionManager` inside the tableViewController. 

This tableview could be reused for participant list drawer, couple audioDeviceSelectionManager inside the tableview doesn't make it reusable for other datasource type.